recipes/orange3-geo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "Orange3-Geo" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "fbe7e008f1dd0766b32bb6a476131351d6bbabada070510d0bf396c4243ae883" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - orange3
+    - scikit-learn
+    - pandas
+    - scipy >=0.17
+    - shapely
+
+test:
+  imports:
+    - orangecontrib.geo
```

Comment:
Orange3-Geo defines [some tests in _setup.py_](https://github.com/biolab/orange3-geo/blob/0371f967bf4059cdae0146c709843045e8d6282e/setup.py#L87), but I don't know how to access it.